test/travis_ci/install_ubuntu_22_04.sh

```diff
@@ -45,3 +45,12 @@ pip3 install "numpy<2.0" mpi4py matplotlib torch
 
 # install data files.
 svn co svn://svn.code.sf.net/p/teca/TECA_data@${TECA_DATA_REVISION} TECA_data
+
+# install TELite library
+git clone https://github.com/LBL-EESA/TELite.git
```

Comment:
could I request that you add installation of TELite to TECA superbuild?  https://github.com/lbl-eesa/teca_superbuild

Reply:
Should I install TELite under `cd TECA_superbuild`?

Reply:
Thanks for creating an issue in teca_superbuild. Sorry, I mas not clear in my question. My question is where I should clone/install TELite in `site/make_install_perlmutter_m1517.sh`?

Reply:
oh I see, yes, TECA_PREFIX